pkgs/applications/audio/musescore/default.nix

```diff
@@ -95,7 +96,14 @@ in stdenv'.mkDerivation (finalAttrs: {
     "--set-default QT_QPA_PLATFORM xcb"
   ];
 
+  preFixup = ''
+    qtWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  dontWrapGApps = true;
+
   nativeBuildInputs = [
+    wrapGAppsHook
```

Comment:
```suggestion
    wrapGAppsNoGuiHook
```

Reply:
I am not sure I understand this change, the problem occurs exactly when a GUI is supposed to be displayed, but the program crashes. In fact, I tested with this change applied and it reverted to the broken behavior.

Can you explain why the no-gui version is expected to work so we can figure out what went wrong?

Reply:
The difference between `wrapGAppsHook3` and `wrapGAppsNoGuiHook` is just the former propagates `gtk3` and `librsvg`, which should not be needed as you just want some environment variables to find schemas. See https://github.com/NixOS/nixpkgs/blob/36316edca39a1f1575613d5ca71bffade3e8a814/pkgs/build-support/setup-hooks/wrap-gapps-hook/default.nix#L20